packages/fxa-shared/metrics/amplitude-event.1.schema.json

```diff
@@ -102,6 +102,36 @@
           "description": "Product ID of a subscription.",
           "type": "string",
           "maxLength": 128
+        },
+        "promotionCode": {
+          "description": "",
```

Comment:
Can you add descriptions for each field here from the Tech Spec?

Reply:
done